data/avgi/avgi ships.txt

```diff
@@ -0,0 +1,2378 @@
+		zoom 0.5
+	"reverse engine" 15 -18
+		angle 15
+		zoom 0.5
+	"steering engine" -15 -18
+		angle 165
+		zoom 0.5
+		left
+	"steering engine" 15 -18
+		angle 195
+		zoom 0.5
+		right
+	"steering engine" -15 3
+		angle 15
+		zoom 0.5
+		right
+	"steering engine" 15 3
+		angle -15
+		zoom 0.5
+		left
+
+	gun 0 -29 "External Launch Tube"
+
+	explode "tiny explosion" 12
+
+	description "The Undulon Orbital Silo was first envisioned as a cheap static defense satellite, little more than a tube capable of storing and launching nuclear-tipped missiles. Over time, this specialization was broadened, and the modern line of Undulons is capable of both mounting a large fixed gun and carrying the energy supply needed to support it. Their mostly hollow construction provides a great deal of internal space for their size, but also makes the drone quite fragile."
+	description "Undulons are usually seen defending critical planets and stations, ready to unload their atomic payloads on any hostile ships that jump into the system."
+	description "	Drones do not come equipped with a hyperdrive. You cannot carry a drone unless you have a ship in your fleet with a drone bay."
+
+
+ship "Optikon"
+	sprite "ship/avgi optikon"
+	thumbnail "thumbnail/avgi optikon"
+	attributes
+		category "Drone"
+		"cost" 54000
+		"hull" 600
+		"automaton" 1
+		"crew equivalent" 1
+		"mass" 30
+		"drag" 1.2
+		"heat dissipation" .9
+		"fuel capacity" 300
+		"outfit space" 60
+		"engine capacity" 24
+		"atmosphere scan" 36
+		"outfit scan power" 42
+		"outfit scan efficiency" 12
+		"cargo scan power" 42
+		"cargo scan efficiency" 12
+		"tactical scan power" 72
+		"asteroid scan power" 36
+		weapon
+			"blast radius" 9
+			"shield damage" 150
+			"hull damage" 90
+			"hit force" 60
+	outfits
+		"Concentrating Solar Cell" 3
+		"Ultracapacitor Cell" 3
+		"Multispectral Scanner"
+		"Spectrometer Array"
+
+		"Magnetoplasma Drive"
+
+	engine 0 21
+	"steering engine" -6 12
+		angle 90
+		left
+	"steering engine" 6 12
+		angle -90
+		right
+
+	explode "tiny explosion" 6
+
+	description "The Optikon Sensor Platform is the latest iteration of a long line of exploration probes developed shortly after the discovery of the hyperdrive. Previously, surveying star systems involved launching thousands of laser-propelled starwisp probes, no larger than a few hundred grams each. The hyperdrive turned this paradigm on its head, allowing the Avgi to build sizable probes designed to quickly map out newly accessible star systems. Over the decades, these gradually shrank as the duty of carrying them between star systems was offloaded onto dedicated exploration vessels, the precursors to the Polari-class Scout. This left the modern Optikon as a highly capable and deployable sensor platform, used in both scientific and combat information roles."
+	description "	Drones do not come equipped with a hyperdrive. You cannot carry a drone unless you have a ship in your fleet with a drone bay."
+
+
+ship "Sonikis"
```

Comment:
This seems likely to end up becoming the drone equivalent of the Boxwing - while the Faes'mar is better in that role, the Sonikis is considerably easier to acquire in large numbers. I don't think it's an issue, though, any more than the Boxwing is an issue.

Reply:
> This seems likely to end up becoming the drone equivalent of the Boxwing - while the Faes'mar is better in that role, the Sonikis is considerably easier to acquire in large numbers. I don't think it's an issue, though, any more than the Boxwing is an issue.

Distinctly agreed, as someone who loves using carriers as freighters. Drone slots are rare, and a full freighter setup will probably find it better to use any drone slots they have on high-damage weapons to scare off raids.